### ISSUE TITLE
Strange behaviour of a validator-engine start up

### ISSUE THREAD

**Issue by Kyborg2011** [ton-blockchain/ton#204]
When I am starting a full node by a command like this:

`validator-engine -d --db /var/ton-work/db/ -C /var/ton-work/etc/ton-global.config.json  -l /var/ton-work/log -S 3600 -v 3 -t 10`

I am getting in `log.thread1.log` errors like that:

```
[ 2][t 1][1575589933.413825274][full-node-shard.cpp:382][!manager]      getnextkey: [Error : 651 : not inited]
[ 2][t 1][1575589946.739737511][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575589949.961083889][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575589956.764132500][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575589967.717857122][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575589978.178229570][full-node-shard.cpp:382][!manager]      getnextkey: [Error : 651 : not inited]
[ 2][t 1][1575589981.986260414][full-node-shard.cpp:382][!manager]      getnextkey: [Error : 651 : not inited]
[ 2][t 1][1575589996.154040813][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575589997.179471254][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575590003.791450262][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575590010.576658964][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575590016.241780996][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
[ 2][t 1][1575590018.332924604][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,388495)]  failed to download state : [Error : 651 : state not found]
```

But when I am changing a parameter `-S`  and increase it in 1000 times:

`validator-engine -d --db /var/ton-work/db/ -C /var/ton-work/etc/ton-global.config.json -l /var/ton-work/log -S 3600000 -v 3 -t 10`

A full node starting good, any errors like "failed to download state" are disappear and all states, blocks and other blockchain information seems to load fine.

I think that it is strange behaviour... Maybe I do not know something important, that can make clear for me such a behaviour of a validator-engine, so I want to understand it. Can someone help me with that?

**Comment by Kyborg2011** [comment]
As I see in docs: "-S: in initial sync download all blocks for last given seconds", so if this parameter == 3600, so I am trying download blocks only for last hour in initial sync. So, maybe an error, that I have found means there is 0 blocks in the last hour?

So, if I am right, all validators, those use a test network in the current time, must increase this parameter (-S) for making initial sync, otherwise there is a big chance to get an error, like I have got "Error : 651 : state not found"?
